test/k8sT/manifests/l7-policy-kafka.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+description: "L7 policy to test HTTP -> Kafka type change"
+metadata:
+  name: "l7-policy"
```

Comment:
Hey, could you change this name to `l7-policy-kafka` please? It's in my ideas to speed a little bit to check the policy enforcement by name, and needs to be the same name as the file without yaml. Thanks.

Reply:
I have to undo this because this test actually needs to test whether we can *change* an existing policy to use a different parser type.